### DISCUSSION TITLE
How can I style hover popups, and can I enter them?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [helix-editor/helix#8134] [Q&A]
They are really hard to see, the color blends with the background and there is no border around them, how can I style them? I tried the `ui.popup.background` and stuff, but doesn't seem to work

Also this is probably for a feature request, but is there no way currently to jump in the hover popover?

**Comment by ath3** [comment]
There is a pr https://github.com/helix-editor/helix/pull/4313 for the popup borders.
To scroll the hover you can use ctrl+u and ctrl+d

**Reply by Nikola-Milovic** [reply]
I am aware about the scrolling it, but sometimes its useful to navigate inside it, especially when there's links involved in the API description.

So the borders still arent here, uf, what about changing the background color, is that possible?
